opencv-stitching-jlink/README.md

```diff
@@ -4,11 +4,11 @@ Sample project for using OpenCV from a generated Java binding made with [Bytedec
 On the command line, run:
 
 ```bash
-$ mvn clean package 
-$ ./target/maven-jlink/bin/stitch panorama_image1.jpg panorama_image2.jpg --output panorama_stitched.jpg
+$ mvn clean package -Djavacpp.platform=linux-x86_64 
```

Comment:
Can you try to use `mvn clean package -Djavacpp.platform.custom -Djavacpp.platform.host` instead like I did for https://github.com/bytedeco/sample-projects/tree/master/opencv-stitching-native?